docs/source/tools/spyglass/spyglass.rst

```diff
@@ -0,0 +1,16 @@
+.. _analysistools-spyglass:
+
+Spyglass
+--------
+
+.. short_description_start
+
+:ref:`analysistools-spyglass` is a framework for building your analysis workflows with a focus on reproducibility and sharing. Spyglass uses community-developed open-source projects such as NWB and DataJoint to manage data in a shareable format, build and run pipelines and store their inputs and outputs in a relational database, and generate visualization of analysis results for sharing with the community. It comes with ready-to-use pipelines for spike sorting, LFP analysis, position processing, and fitting of state space models for decoding variables of interest from neural data.
+:bdg-link-primary:`Docs <https://lorenfranklab.github.io/spyglass/0.4/>`
```

Comment:
```suggestion
:bdg-link-primary:`Docs <https://lorenfranklab.github.io/spyglass/>`
```

Reply:
It would be better to link to the homepage for the documentation rather than version 0.4.